addOns/ascanrulesBeta/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Changed
+- The .env file scan rule now performs even better content checks. (Issue 6629)
```

Comment:
We typically add new items at the end of a list not the begining, but not a biggie...

Reply:
It can actually update the existing change.

Reply:
as in like put this in the bottom of the changed section?

Reply:
Update the entry `The .env file scan rule now performs a simple content check to reduce false positives`, does not make sense having both.

Reply:
Yes, they are both about the same changes (both issues can still be referenced).

Reply:
set

Reply:
Thanks, please excuse my confusing message. I must have been working on my phone and not proof read 😢